app/views/static_pages/community_rules.html.erb

```diff
@@ -35,7 +35,7 @@
         <strong>Asking about or encouraging the use of Windows</strong>
       </li>
       <li>
-        <strong>Discussing mental health issues in depth or seeking mental health help from other users</strong>
+        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic remedies, drugs, etc.
```

Comment:
``` suggestion
        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic remedies, medicinal remedies or advise, etc.
```
- removed drugs. 
- Also included medicine. Rachel requested this https://github.com/TheOdinProject/moderator/issues/2#issue-1393455684 and I think it belongs here

Reply:
I decided to not use the word "advise" because I believe giving or asking for advice is included in "discussing" one of these topics.